sleap/gui/dialogs/importvideos.py

```diff
@@ -91,7 +95,7 @@ class ImportParamDialog(QDialog):
         filenames (list): List of files we want to import.
     """
 
-    def __init__(self, filenames: List[str], *args, **kwargs):
+    def __init__(self, filenames: List[str], messages: dict = dict(), *args, **kwargs):
```

Comment:
Big no-no here! Mutable types should not be instantiated as function defaults! More info: https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments

See also the [typing module docs](https://docs.python.org/3.7/library/typing.html) for the proper way to typehint `dict`s in python 3.7.

What we want here is something like:
```
    def __init__(self, filenames: List[str], messages: Optional[Dict[str, str]] = None, *args, **kwargs):
        if messages is None:
            messages = dict()
```

Reply:
Got it! Thanks